docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -40,6 +43,8 @@ See the appropriate section for your hypervisor and operating system:
 # chmod +x /usr/local/bin/docker-machine-driver-kvm
 ----
 
+After you have set up the binary, proceed to instructions for your linux distribution.
```

Comment:
Linux should be capitalized, and we can avoid comma splicing by rearranging the sentence:

```suggestion
Proceed to the instructions for your Linux distribution after you have set up the binary.
```